### DISCUSSION TITLE
Edit Page form contents disappearing after switching to relationship tab

### DISCUSSION THREAD

**Discussion by apvanlaan** [filamentphp/filament#12608] [Help]
### Package

Form builder

### Package Version

v3

### How can we help you?

So I was following the docs here : https://filamentphp.com/docs/3.x/panels/resources/relation-managers#combining-the-relation-manager-tabs-with-the-form because I wanted the relationships on my resource to be tabbed instead of falling to the end of the form.  What I'm experiencing is that if I click over to one of the relationship tabs, then go back to the Edit tab for my form, all of the data is gone from my form (all fields are blank).  I'm just wondering if this is a known behavior or if I've missed some step in working with the Relation Manager.  Thanks in advance!

**Comment by leandrocfe** [comment]
If you are following the docs, this was supposed to work. Maybe you could provide a mini repo on Github to reproduce this issue (you can also check the Filament issues tab on Github)

**Comment by otavio-araujo** [comment]
Did you get a solution??
